src/query_results.js

```diff
@@ -21,7 +21,7 @@ const debug = require('debug')('bte:biothings-explorer-trapi:QueryResult');
  *
  * @typedef {string} QueryEdgeID
  *
- * @typedef {Object.<string, EdgeData>} DataByEdge
+ * @typedef {Object.<string, EdgeData>} recordsByQEdgeID
```

Comment:
Can you double check the JSDoc syntax? I thought when you define a typedef, it's supposed to be TitleCase, and then when you use it elsewhere, it can be like `recordsByQEdgeID: RecordsByQEdgeID`. But it's been awhile since I've looked it up.

Reply:
The docs have some examples here: https://jsdoc.app/tags-typedef.html
So I think you have it correct now:
`@typedef {Object.<string, EdgeData>} RecordsByQEdgeID`
`@param {RecordsByQEdgeID} recordsByQEdgeID`